modules/sharedaddy/sharing.php

```diff
@@ -331,10 +339,10 @@ public function management_page() {
 						<th scope="row"><label><?php _e( 'Button style', 'jetpack' ); ?></label></th>
 						<td>
 							<select name="button_style" id="button_style">
-								<option<?php if ( $global['button_style'] == 'icon-text' ) echo ' selected="selected"';?> value="icon-text"><?php _e( 'Icon + text', 'jetpack' ); ?></option>
-								<option<?php if ( $global['button_style'] == 'icon' ) echo ' selected="selected"';?> value="icon"><?php _e( 'Icon only', 'jetpack' ); ?></option>
-								<option<?php if ( $global['button_style'] == 'text' ) echo ' selected="selected"';?> value="text"><?php _e( 'Text only', 'jetpack' ); ?></option>
-								<option<?php if ( $global['button_style'] == 'official' ) echo ' selected="selected"';?> value="official"><?php _e( 'Official buttons', 'jetpack' ); ?></option>
+								<option<?php if ( $global['button_style'] == 'icon-text' ) { echo ' selected="selected"';}?> value="icon-text"><?php _e( 'Icon + text', 'jetpack' ); ?></option>
```

Comment:
same as above, in this context, one liner conditional makes it easy to read, this is a gist when it converted into multiple lines https://gist.github.com/ysupr/c02065f3af3069ea3ee11212485bfe94 if